docs/transition-phase-2.md

```diff
@@ -0,0 +1,107 @@
+# Transition your stake from phase-1 to phase-2 through CLI
+
+## Understanding the transition
+
+The transition from phase-1 to phase-2 is a manual process that requires the 
+staker to transition their stake to phase-2 of the Babylon network. 
+This transition has the option of being executed through the `stakercli` binary 
+and this guide will show you how to do it through the `stakercli` binary. For 
+detailed information about stake eligibility 
+criteria, registration requirements, and how to gain voting power and earn 
+rewards in phase-2, please refer to the 
+[Transition of Existing Stakes to the Babylon Chain](https://github.com/babylonlabs-io/babylon/blob/main/docs/stake-registration.md).
+<!-- TODO: Update link -->
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Setup](#setup)
+- [Transition your stake](#transition-your-stake)
+- [Verify the transition](#verify-the-transition)
+
+## Prerequisites
+
+To transition your stake from phase-1 to phase-2, you need to have the 
+following:
+- Access to your existing Bitcoin wallet from phase-1 (needed to sign transactions 
+  and prove ownership of staked funds).
+- The Bitcoin block height in which your staking transaction was included, which 
+  will be used in the transition command.
+- A funded Babylon address, which will be used to receive the rewards from the 
+  transition.
+- Global parameter file as seen here https://github.com/babylonlabs-io/networks/blob/main/bbn-test-5/parameters/global-params.json 
+  that were used in phase-1. This will be used in the transition command later 
+  on.
+- A running instance of the `stakercli` binary. Please follow the `stakercli` 
+  [setup guide](../README.md#3-btc-staker-installation) to complete the setup of 
+  the `stakercli` with your keys before proceeding.
+
+## Setup 
+
+First, you will need to dump the default configuration file. This creates a 
+template configuration file that you'll customize with your specific values 
```

Comment:
The configuration file only has network details and wallet credentials so this sentence `(like your Bitcoin block height that your staking transaction was included in,` shouldn't be here imo